tests/foreman/virtwho/ui/test_esx.py

```diff
@@ -54,6 +57,14 @@ def form_data():
 
 
 class TestVirtwhoConfigforEsx:
+    @pytest.mark.tier2
+    def setup_method(self):
+        """Set up a clean env for tests."""
+        virtwho_cleanup()
+        guest_name, guest_uuid = get_guest_info(settings.virtwho.esx.hypervisor_type)
+        if Host.list({'search': guest_name}):
+            Host.delete({'name': guest_name})
+
     @pytest.mark.tier2
```

Comment:
Remove all of these markers if you're going to apply tier2 mark at the class level please.

Reply:
> Remove all of these markers if you're going to apply tier2 mark at the class level please.

Hey,  @mshriver do you mean remove all the  @pytest.mark.tier2 before test_XXX method, and put it only before class "class TestVirtwhoConfigforEsx"?